src/tairzset.c

```diff
@@ -1722,7 +1728,99 @@ int exZpopMinMaxGenericCommand(RedisModuleCtx *ctx, RedisModuleString **argv, in
         return REDISMODULE_OK;
     }
 
-    exGenericZpopCommand(ctx, key, where, count); 
+    exGenericZpopCommand(ctx, key, where, NULL, count); 
+
+    return REDISMODULE_OK;
+}
+
+typedef struct exBzpopInfo {
+    int where;
+} ExBzpopInfo;
+
+int exBzpopReplyCallback(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+    RedisModuleString *key_str = RedisModule_GetBlockedClientReadyKey(ctx);
+    RedisModuleKey *key = RedisModule_OpenKey(ctx, key_str, REDISMODULE_WRITE);
+    int type = RedisModule_KeyType(key);
```

Comment:
I think you should close the key manually or you will get mem leak.

Reply:
How about use `RedisModule_AutoMemory(ctx);`